halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/EditOAuth2Command.java

```diff
@@ -74,6 +82,10 @@ protected AuthnMethod editAuthnMethod(OAuth2 authnMethod) {
 
     authnMethod.setProvider(provider != null ? provider : authnMethod.getProvider());
 
+    if (!userInfoRequirements.isEmpty()) {
```

Comment:
By supplying the default `new OAuth2.UserInfoRequirements()` to this parameter, how will this be empty?